bedrock/firefox/tests/test_base.py

```diff
@@ -606,7 +606,7 @@ def test_fx_firstrun_40_0(self, render_mock):
         req = self.rf.get('/en-US/firefox/firstrun/')
         self.view(req, version='40.0')
         template = render_mock.call_args[0][1]
-        eq_(template, ['firefox/australis/fx40/firstrun.html'])
+        eq_(template, ['firefox/firstrun/firstrun.html'])
```

Comment:
So the default template has not changed to the one above so, the test fails if I do not change the template to the new one.